src/styles/blocks.scss/logo.scss

```diff
@@ -0,0 +1,15 @@
+.logo {
+  &__img {
+    height: 30px;
```

Comment:
Add transition to `&__img`.

And check all over the project that you have transition with property and duration for the element.